src/App.tsx

```diff
@@ -15,58 +15,83 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type Good = (typeof goods)[number];
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+export class App extends React.Component {
+  state = {
+    selectedGood: 'Jam',
+  };
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
+  handleGoodSelect = (good: Good) => {
+    this.setState({ selectedGood: good });
+  };
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+  handleClearSelection = () => {
+    this.setState({ selectedGood: '' });
+  };
 
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
+  addButton = (good: Good) => (
+    <button
+      data-cy="AddButton"
+      type="button"
+      className="button"
+      onClick={() => this.handleGoodSelect(good)}
+    >
+      +
+    </button>
+  );
+
+  removeButton = (
+    <button
+      data-cy="RemoveButton"
+      type="button"
+      className="button is-info"
+      onClick={() => this.handleClearSelection()}
```

Comment:
If your method don't get any argument, you don't need to create new arrow function. Just pass your method. [[checklist]](https://github.com/mate-academy/react_goods-selector/blob/master/checklist.md)